src/potions.ts

```diff
@@ -274,7 +276,7 @@ function useAsValuable(potion: Potion, embezzlers: number, embezzlersOnly: boole
   const value = potion.value(embezzlers);
   const price = potion.price(false);
   const amountsAcquired = value.map((value) =>
-    (!embezzlersOnly || value.name === "embezzlers") && value.value - price > 0
+    (!embezzlersOnly || value.name === "embezzler") && value.value - price > 0
       ? acquire(value.quantity, potion.potion, value.value, false)
```

Comment:
I am always concerned about things like `value.value` being written down--I think we can declare cleanup out of scope, but I would like to try to make this a little more legible for future generations in the future.

Reply:
agreed